Files/View Models/SelectedItemsPropertiesViewModel.cs

```diff
@@ -82,110 +92,143 @@ public bool IsItemSelected
-                    else if (App.CurrentInstance.ContentPage.SelectedItem.PrimaryItemAttribute == StorageItemTypes.File)
-                    {
-                        dateCreated = StorageFile.GetFileFromPathAsync(App.CurrentInstance.ContentPage.SelectedItem.ItemPath).GetResults().DateCreated;
-                    }
-                    return ListedItem.GetFriendlyDate(dateCreated);
-                }
-                else
-                {
-                    return null;
-                }
-            }
+            get => _ItemCreatedTimestamp;
+            set => Set(ref _ItemCreatedTimestamp, value);
         }
-
-        public string ItemModifiedTimestamp
+        public string _ItemAccessedTimestamp;
+        public string ItemAccessedTimestamp
         {
-            get
-            {
-                if (App.CurrentInstance.ContentPage == null) return null;
-
-                if (App.CurrentInstance.ContentPage.IsItemSelected)
-                {
-                    return App.CurrentInstance.ContentPage.SelectedItem.ItemDateModified;
-                }
-                else
-                {
-                    return null;
-                }
-            }
+            get => _ItemAccessedTimestamp;
+            set => Set(ref _ItemAccessedTimestamp, value);
         }
-
-        public ImageSource FileIconSource
+        public string _ItemFileOwner;
+        public string ItemFileOwner
         {
-            get
-            {
-                if (App.CurrentInstance.ContentPage == null) return null;
+            get => _ItemFileOwner;
+            set => Set(ref _ItemFileOwner, value);
+        }
+        public string _ItemMD5Hash;
+        public string ItemMD5Hash
+        {
+            get => _ItemMD5Hash;
+            set => Set(ref _ItemMD5Hash, value);
+        }
+        public Visibility _ItemMD5HashVisibility;
 
-                if (App.CurrentInstance.ContentPage.IsItemSelected)
-                {
-                    return App.CurrentInstance.ContentPage.SelectedItem.FileImage;
-                }
-                else
-                {
-                    return null;
-                }
-            }
+        public Visibility ItemMD5HashVisibility
+        {
+            get => _ItemMD5HashVisibility;
+            set => Set(ref _ItemMD5HashVisibility, value);
         }
+        public Visibility _ItemMD5HashProgressVisibiity;
 
-        public bool LoadFolderGlyph
+        public Visibility ItemMD5HashProgressVisibility
         {
-            get
+            get => _ItemMD5HashProgressVisibiity;
+            set => Set(ref _ItemMD5HashProgressVisibiity, value);
+        }
+        #endregion
+        #region Properties
+        public ProgressBar ItemMD5HashProgress
+        {
+            get; set;
+        }
+        public ListedItem Item { get; }
+        public Visibility FileOwnerVisibility { get => App.AppSettings.ShowFileOwner ? Visibility.Visible : Visibility.Collapsed; }
```

Comment:
Why don't we just bind the TextBlock in the property dialogs xaml to this setting instead of checking it in the view model?

Reply:
Resolved [here](https://github.com/files-community/files-uwp/pull/1055/commits/ceaea013f1dc474a46170806cd0314f934bde7c2)